page.md

```diff
@@ -8,19 +8,26 @@ representing a PAGE document](https://ocr-d.github.io/mets#media-type-for-page-x
 
 ## URL for imageFilename / filename
 
-The `imageFilename` of the `<pg:Page>` and `filename` of the `<pg:AlternativeImage>` element MUST be a URL. A local filename should be a `file://` URL.
+The `imageFilename` of the `<pg:Page>` and `filename` of the
+`<pg:AlternativeImage>` element MUST be a URL. A local filename should be a
+`file://` URL.
 
-All URL used in `imageFilename` and `filename` [MUST be referenced in a fileGrp in METS](https://ocr-d.github.io/mets#if-in-page-then-in-mets).
+All URL used in `imageFilename` and `filename` [MUST be referenced in a fileGrp
+in METS](https://ocr-d.github.io/mets#if-in-page-then-in-mets).
 
 ## Original image as imageFilename
 
-The `imageFilename` attribute of the `<pg:Page>` MUST reference the original image and MUST NOT change between processing steps.
+The `imageFilename` attribute of the `<pg:Page>` MUST reference the original
+image and MUST NOT change between processing steps.
 
 ## AlternativeImage for derived images
 
-To encode images derived from the original image, the `<pc:AlternativeImage>` should be used. Its `filename` attribute should reference the URL of the derived image.
+To encode images derived from the original image, the `<pc:AlternativeImage>`
```

Comment:
"should" or "MUST"?

Reply:
MUST

Reply:
Ah, now I remember why it is `SHOULD` and not `MUST`: We do not use this mechanism at the moment. Rather we explicitly demand input file groups on call to a processor. So there is little benefit in setting it in the PAGE XML file.

If we demand it, we must implement it...